src/index.js

```diff
@@ -0,0 +1,257 @@
+    form.addEventListener('submit', async (e) => {
+      e.preventDefault();
+
+      const inputName = e.target.closest('form').children[0];
+      const inputStartDate = e.target.closest('form').children[1];
+      const inputEndDate = e.target.closest('form').children[1];
+
+      const commentsContainer = e.target.closest('.popup_main').children[3];
+
+      let html = '';
+
+      html += `<p> ${inputStartDate.value} - ${inputEndDate.value} by ${inputName.value}</p>`;
+
+      commentsContainer.insertAdjacentHTML('beforeend', html);
+
+      const { id } = e.target.closest('.popup_container');
+
+      createReservation(projectID, id, inputName.value, inputStartDate.value, inputEndDate.value);
+    });
+  });
+};
+const createCards = async () => {
+  try {
+    let html = '';
+
+    const items = await getData();
+    const { categories } = items;
+
+    const likeID = await getLikes(projectID);
+
+    categories.forEach((category) => {
+      html += `<div id = ${category.idCategory} class="item_contianer">
+                <img src="${category.strCategoryThumb}" alt="">
+                <div class="name_like_conatiner">
+                    <p>${category.strCategory}</p> 
+                    
+                    <div class="likes_heart"> 
+                    <p>${likesShow(likeID, category) ? likesShow(likeID, category) : '0'}</p>
+                    <i class="fas fa-heart"></i>
+                    </div>
+                </div>
+                <button class="comments popup_btn">Comments</button>
+                <button class="reservation popup_btn">Reservation</button>
+            </div>`;
+    });
+
+    mainItemsContainer.insertAdjacentHTML('afterbegin', html);
+
+    comments = document.querySelectorAll('.comments');
+    reservation = document.querySelectorAll('.reservation');
+    const heartBtn = document.querySelectorAll('.fa-heart');
+
+    comments.forEach((comment, i) => {
+      comment.addEventListener('click', (e) => {
+        const { id } = e.target.closest('.item_contianer');
+
+        const [filteredObj] = categories.filter((category) => category.idCategory === id);
+        createPopupWindow(filteredObj, i + 1);
+      });
+    });
+
+    reservation.forEach((reserve, i) => {
+      reserve.addEventListener('click', (e) => {
+        const { id } = e.target.closest('.item_contianer');
+        const [filteredObj] = categories.filter((category) => category.idCategory === id);
+        createPopupWindowReservation(filteredObj, i + 1);
+      });
+    });
+
+    heartBtn.forEach((heart) => {
+      heart.addEventListener('click', (e) => {
+        const { id } = e.target.closest('.item_contianer');
+        heart.classList.add('colour_red');
+        let likeCounter = e.target.previousElementSibling.textContent;
+        likeCounter = +likeCounter;
+        likeCounter += 1;
+        e.target.previousElementSibling.textContent = String(likeCounter);
+        postLikes(id, projectID);
+      });
```

Comment:
- [x]  OPTIONAL
When I continue to click on the like button, it keeps the counter only keeps increasing. However, I think you can improve upon your _likes_ feature this way:
    - Like icon should toggle to indicate _like_ or _unlike_ when a particular user clicks on the like icon.
    - Total count of likes per user should be 1.

Can you try achieving this with the help of local storage?